test/common/schemaform/save-in-progress/actions.unit.spec.js

```diff
@@ -312,7 +339,7 @@ describe('Schemaform save / load actions:', () => {
         status: 404
       }));
 
-      return thunk(dispatch, getState).then(() => {
+      return thunk(dispatch, getState).catch(() => {
```

Comment:
This seems to be kind of a big change for it being only one word. What are the ramifications? (I didn't quite catch what made this change necessary.)

Reply:
These are necessary as the logic they are testing is also within a `catch`, I'm not sure why this wasn't required before.